react/form/src/i18n/index.js

```diff
@@ -0,0 +1,14 @@
+import * as Yup from 'yup';
+import i18next from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import locales from 'i18n/locales';
+import i18nextOptions from 'i18n/i18nextOptions';
+import yupLocale from 'i18n/yupLocale';
+
+Yup.setLocale(yupLocale);
+
+i18next.use(initReactI18next).init(i18nextOptions);
+
+Object.keys(locales).forEach(locale => {
+  i18next.addResourceBundle(locale, i18nextOptions.ns[0], locales[locale]);
```

Comment:
we are not using the `resources` approach?

Reply:
Updated using the resources approach (but without the `translation` key in the json locales) and refactoring some stuff.